charts/datagovuk/templates/_find.tpl

```diff
@@ -2,14 +2,16 @@
 {{- $environment := eq .Values.environment "test" | ternary "development" .Values.environment -}}
 - name: CKAN_DOMAIN
   value: {{ .Values.find.config.ckanDomain }}
-- name: GOVUK_APP_DOMAIN
-  value: "www.gov.uk"
 - name: ES_HOST
   value: http://{{ $.Release.Name }}-opensearch
-- name: RAILS_ENV
-  value: {{ $environment }}
 - name: ES_INDEX
   value: datasets-{{ $environment }}
+- name: GOVUK_APP_DOMAIN
+  value: "www.gov.uk"
```

Comment:
I'm a little dubious about this value for GOVUK_APP_DOMAIN, but by all means try it! (Maybe Find uses this differently to the other apps? I haven't looked.)

Reply:
It seems like only Prod needs it, other envs don't need it so will try and understand why later

Reply:
One thing that comes to mind based on that: what's the value of RAILS_ENV in staging/integration? (you want it to be `production` in all three)